assets/promptflow/models/bring-your-own-data-chat-qna/model.yaml

```diff
@@ -1,7 +1,7 @@
 path:
-  container_name: models
-  container_path: samples/bring_your_own_data_chat_qna/v6/model_folder
-  storage_name: promptflowsamples
+  container_name: gallery-samples
+  container_path: bring_your_own_data_chat_qna
```

Comment:
The container_path doesn't contain version info. If we found this version example has issue. Do we have place to find the previous working version ?

Reply:
These examples are uploaded to this storage account by a release pipeline, from an ADO repo. Can we not use that as the version control mechanism here, instead of creating multiple versions within the storage account? Git retains history of changes in a more granular fashion, and will allow us to perform reverts more effectively if needed.

Perhaps I'm missing something, but what benefit are we gaining by versioning the samples in storage?